$OpenBSD: patch-gas_config_tc-msp430_c,v 1.1 2011/05/09 08:58:56 jasper Exp $

Fix for SF 3293911
gas 2.21 segfault when building gcc 4.4.5

From upstream git:
mspgcc/binutils f3f850212509bd6a19bed10e1d5783cc5f95777a

--- gas/config/tc-msp430.c.orig	Sun May  8 22:51:54 2011
+++ gas/config/tc-msp430.c	Sun May  8 22:52:12 2011
@@ -661,7 +661,7 @@ get_operand (void)
   char* dp;
   int operand_length = 0;
 
-  while (sp && ',' != *sp && ';' != *sp && '\n' != *sp)
+  while (*sp && ',' != *sp && ';' != *sp && '\n' != *sp)
     {
       if (! ISSPACE(*sp))
 	++operand_length;
